src/WattleScript.Interpreter/Tree/Expressions/BinaryOperatorExpression.cs

```diff
@@ -69,7 +69,7 @@ class LinkedList
 		const Operator NIL_COAL_ASSIGN = Operator.NilCoalescing;
 		const Operator SHIFTS = Operator.BitLShift | Operator.BitRShiftA | Operator.BitRShiftL;
 		const Operator NIL_COAL_INVERSE = Operator.NilCoalescingInverse;
-		const Operator RANGES = Operator.InclusiveRange | Operator.ExclusiveRange | Operator.LeftExclusiveRange | Operator.RightExclusiveRange;
+		internal const Operator RANGES = Operator.InclusiveRange | Operator.ExclusiveRange | Operator.LeftExclusiveRange | Operator.RightExclusiveRange;
```

Comment:
Instead of exposing all these fields, better to add a method that does the check if it's a range constructor, then return the two expressions. 

`bool TryGetRange(out Expression min, out Expression max, out Operator op)`

Reply:
Did that somehow differently but `internal` is removed. See 8e870ee